tests/unit/test_taxonomy.py

```diff
@@ -136,13 +136,15 @@ def test_create_brand_taxonomy_mapping(self):
                 "en:arev": {"name": {"en": "Arèv"}},
                 "en:arrighi": {"name": {"en": "Arrighi"}},
                 "en:voiles-au-vent": {"name": {"en": "Voiles au Vent"}},
+                "xx:turini": {"name": {"xx": "Turini"}},
```

Comment:
should we add a test "without prefix" ? or is it something not possible anyway ?

Reply:
This is not possible, every entry has a prefix.

Reply:
But I'll add one with eg. fr:auchan to verify indeed we get auchan !

Reply:
ok, because your other change made it look like it was possible :eyes:

Reply:
could be simplified to just
```
mapping[unprefixed_key] = node.names.get("xx", node.names["en"])
```

Reply:
I prefer avoiding an early exception…